resources/js/src/app/components/itemList/ItemStoreSpecial.js

```diff
@@ -77,8 +78,18 @@ Vue.component("item-store-special", {
 
         getPercentageSale()
         {
-            // eslint-disable-next-line
-            let percent = (1 - this.variationRetailPrice.unitPrice.value / this.recommendedRetailPrice.unitPrice.value ) * -100;
+            let percent;
+
+            if (this.specialOfferPrices !== null)
```

Comment:
Anstatt !== null zu verwenden, könntest du hier die Funktion isDefined(object) verwenden, aus utils.js.
In der 1. Zeile muss dafür noch die jeweilige Funktion importiert werden.

Reply:
done